src/me/corriekay/pokegoutil/windows/PokemonTab.java

```diff
@@ -816,7 +816,8 @@ private void selectLessThanIv() {
         System.out.println("Selecting Pokemon with IV less than: " + ivTransfer.getText());
 
         for (int i = 0; i < pt.getRowCount(); i++) {
-            final double pIv = Double.parseDouble((String) pt.getValueAt(i, 3));
+            final double pIv = (double) pt.getValueAt(i, 5) * 100;
```

Comment:
I am quite sure the `5` here means the id of the column, right?
First of, we have the policy to don't use magic numbers, but use constants, or at least variables with a explaining name for it.

But here this shouldn't be the fix. columnIds are really not hard-coded. Columns are moveable, etc etc. That's why I've implemented them to be auto-generated.
So to get the ID of the IV column you have to use `PokeColumn.IV_RATING.id`.

Is that understandable?

Or have I missed something in your code? xD